src/gui/MixerView.cpp

```diff
@@ -211,14 +214,18 @@ void MixerView::refreshDisplay()
 		m_racksLayout->removeWidget(mixerChannelView->m_effectRackView);
 
 		delete mixerChannelView;
+
+		// We are only deleting the views but not the actual mixer channels.
+		// Therefore we do not have to disconnect from the solo/mute models.
 	}
 	m_channelAreaWidget->adjustSize();
 
 	// re-add the views
-	m_mixerChannelViews.resize(Engine::mixer()->numChannels());
+	m_mixerChannelViews.resize(getMixer()->numChannels());
 	for (int i = 1; i < m_mixerChannelViews.size(); ++i)
 	{
 		m_mixerChannelViews[i] = new MixerChannelView(m_channelAreaWidget, this, i);
+		// We are readding the views for existing channels so we do not have to connect to solo/mute here
```

Comment:
I believe `refreshDisplay` is also called when we load the project, so maybe the connections are missing here.

Reply:
Argh, you're right. Of all the cases I did not check the one this issue is actually about. :sweat_smile:

Fixed with commit f01cf366338.